src/relay.py

```diff
@@ -0,0 +1,165 @@
+'''
+Controlling the 4 relay channels with the raspberry pi zero w
+RPi is connected to 4 relay channels on GPIO26, GPIO19, GPIO13, GPIO06 (in this order)
+
+
+>>>>>>> 892991e028f6eaf4ae6bd0735b3b875f3d701d80
+
+Author: Carson Berry
+Date: February 2, 2021
+
+Inputs: null 
+
+Outputs: zero if successfull 
+
+Usage: 
+import relay
+relay.init_one()
+relay.on1()
+relay.off1()
+relay.demo()
+
+
+'''
+
+import RPi.GPIO as GPIO
+import time
+
+
+# set all the pins to output pins
+
+GPIO.setmode(GPIO.BCM) # GPIO Assign mode so that the numbers below are the GPIO assigned names
+in1 = 26
+in2 = 19
+in3 = 13
+in4 = 6
+
+
+def init_one():
```

Comment:
Can also be turned into generic function, `init(input pin. GPIO.OUT)`. You can define pins as constants as part of the design to more easily change later.